common/constants.ts

```diff
@@ -204,7 +204,109 @@ export enum WAZUH_MODULES_ID{
   TSC = 'tsc',
   CIS_CAT = 'ciscat',
   VIRUSTOTAL = 'virustotal',
-  GDPR = 'gdpr'
+  GDPR = 'gdpr',
+}
+
+export enum WAZUH_MODULES_CY_TEST_ID {
```

Comment:
It's not scalable, and this file is for business constants. 
I think this kind of constants should be in a different type of files, I suggest using a `<same-name-that-component>.cy.ts` file maybe for each tsx file or common resources.

Reply:
Why we need these values defined in a file? There are unique, why not defined directly in the components?

Reply:
@frankeros I agree with you, it seems like a good idea to pass the constants corresponding to test in a different file.

Reply:
@Desvelao It seemed like a good idea to leave these constants outside the component, I tried to follow the idea that was used for the constants in 'modules'